podcasts/New Detail/Playlist Header/PlaylistHeaderView.swift

```diff
@@ -0,0 +1,100 @@
+import SwiftUI
+
+struct PlaylistHeaderView: View {
+    @EnvironmentObject var theme: Theme
+    @StateObject var viewModel: PlaylistDetailViewModel
+
+    var body: some View {
+        HStack(alignment: .top, spacing: 0) {
+            VStack {
+                HStack {
+                    Spacer()
+                    PlaylistArtworkView(items: viewModel.images, imageSize: 192)
+                        .frame(width: 192.0, height: 192.0)
+                        .padding(.top, 5.0)
+                        .shadow(color: .black.opacity(0.2), radius: 30, x: 0, y: 2)
+                    Spacer()
+                }
+
+                VStack(spacing: 10.0) {
+                    Text(viewModel.playlist.playlistName)
+                        .font(size: 22.0, style: .body, weight: .bold)
```

Comment:
Should this be base on the `.body` style or something larger? like `.title` ?

Reply:
I use it too, but I do think the style you select influence the way the sizing is done when changing font sizes in the system.
I normally try to align the style to the size the system uses  for the default Text Size.
You can check the values here: https://developer.apple.com/design/human-interface-guidelines/typography#Specifications

Reply:
@SergioEstevao  Do you assign the style only or event the font size?

Reply:
I do both, style and size